python/cugraph/pregel/traversal/bfs_bsp.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import cudf
+from collections import OrderedDict
+
+
+def bfs_df_pregel(df, start, src_col='src', dst_col='dst', copy_data=True):
+    """
+    This function executes an unwieghted Breadth-First-Search (BFS) traversal
+    to find the distances and predecessors from a specified starting vertex
+
+    NOTE: Only reachable vertices are returned
+    NOTE: data is not sorted
```

Comment:
stored -> copied ?

Reply:
not sure the question?

Reply:
About: `NOTE: data is not sorted` I wondered if you meant `NOTE: data is not duplicated/copied`